autoria.py

```diff
@@ -0,0 +1,242 @@
+"""Python auto.ria.com API.
+
+Python implementation of API intended for calulating
+average used car prices that are sold on http://auto.ria.com
+"""
+
+
+import requests
+import json
+from fnmatch import fnmatch
+
+
+def make_request(url: str) -> json:
+    """Send get request and return data in JSON.
+
+    Args:
+        url - string with url returning needed
+        data (categories, bodystyles etc.)
+
+    Returns:
+        JSON-formatted response text.
+    """
+    response = requests.get(url)
+    if response.status_code == 200:
+        return json.loads(response.text)
+    else:
+        print('Ups! Something went wrong in processing of {}'.format(url))
+
+
+def get_categories() -> json:
+    """Get available vehicle types from auto.ria.com.
+
+    No args needed because the list has fixed url.
+
+    Returns:
+        JSON-formatted list of pairs ''name: value'' like:
+        [
+            {
+            "name": "Легковые",
+            "value": 1
+            },
+            ...
+        ]
+    """
+    url = 'http://api.auto.ria.com/categories'
+    return make_request(url)
+
+
+def get_bodystyles(category: int) -> json:
+    """Get available bodystyles from auto.ria.com.
+
+    Args:
+        category - integer with needed category id.
+    Returns:
+        JSON-formatted list of pairs ''name: value'' like:
+        [
+            {
+            "name": "Седан",
+            "value": 3
+            },
+            ...
+        ]
+    """
+    url = 'http://api.auto.ria.com/categories/{}/bodystyles' \
+          .format(str(category))
+    return make_request(url)
+
+
+def get_marks(category: int) -> json:
+    """Get available car marks from auto.ria.com.
+
+    Args:
+        category - integer with needed category id.
+
+    Returns:
+        JSON-formatted list of pairs ''name: value'' like:
+        [
+            {
+            "name": "Acura",
+            "value": 98
+            },
+            ...
+        ]
+    """
+    url = 'http://api.auto.ria.com/categories/{}/marks' \
+          .format(str(category))
+    return make_request(url)
+
+
+def get_models(category: int, mark: int) -> json:
+    """Get available models for selected mark from auto.ria.com.
+
+    Args:
+        category - integer with needed category id.
+        mark - integer with needed mark id
+            (since we're getting models of the mark)
+
+    Returns:
+        JSON-formatted list of pairs ''name: value'' like:
+        [
+            {
+            "name": "CL",
+            "value": 953
+            },
+            ...
+        ]
+    """
+    url = 'http://api.auto.ria.com/categories/{}/marks/{}/models' \
+          .format(str(category), str(mark))
+    return make_request(url)
+
+
+def get_states() -> json:
+    """Get available states from auto.ria.com.
+
+    Args aren't needed because the url is fixed.
```

Comment:
I think it is not necessary to mention why args are not needed, just specify "Returns:" and that's it.